include/doocsServerTestHelper.h

```diff
@@ -276,16 +291,16 @@ std::vector<TYPE> DoocsServerTestHelper::doocsGetArray(const std::string& name)
         val.push_back(res.get_long(i));
       }
     }
-  }
-  else if(std::is_floating_point<TYPE>()) {
-    for(int i = 0; i < res.length(); i++) {
-      val.push_back(res.get_float(i));
     }
```

Comment:
The view is not nice because the the diff does not find matching lines, but line 293 and 294 seem to have a closing curly bracket at the same indentation level.

Reply:
fixed